FeatureEngine.py

```diff
@@ -312,27 +322,17 @@ def is_hori_aligned(t1, t2, moe):
                 if t.contains_digit and dist < features["dist_nearest_cell_w_digit"]:
                     features["dist_nearest_cell_w_digit"] = dist
 
-        features["rel_dist_nearest_cell_w_date"] = (
-            features["dist_nearest_cell_w_date"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_currency"] = (
-            features["dist_nearest_cell_w_currency"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_address"] = (
-            features["dist_nearest_cell_w_address"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_datelabel"] = (
-            features["dist_nearest_cell_w_datelabel"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_numlabel"] = (
-            features["dist_nearest_cell_w_numlabel"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_totallabel"] = (
-            features["dist_nearest_cell_w_totallabel"] / invoice_diag
-        )
-        features["rel_dist_nearest_cell_w_digit"] = (
-            features["dist_nearest_cell_w_digit"] / invoice_diag
-        )
+        for feature in ["dist_nearest_cell_w_date","dist_nearest_cell_w_currency","dist_nearest_cell_w_address","dist_nearest_cell_w_datelabel","dist_nearest_cell_w_numlabel","dist_nearest_cell_w_totallabel","dist_nearest_cell_w_digit"]:
```

Comment:
this seems like a workaround. I can set the default to 0.75. Also why 0.75?

Reply:
It's an arbitrary number, I should probably use a variable here. Setting the default to 0.75 wouldn't work because there could be distances that are larger than 0.75 times of the diagonal distance. Because we update by reducing it each time we see a smaller number, these distances would not be reflected